apache2/msc_multipart.c

```diff
@@ -130,7 +134,41 @@ static int multipart_parse_content_disposition(modsec_rec *msr, char *c_d_value)
          */
 
         char quote = '\0';
-        if ((*p == '"') || (*p == '\'')) {
+        if (name == "filename*") {
```

Comment:
```suggestion
        if (strcmp(name, "filename*") == 0) {
```
This is the way it is used in the other cases, so I guess it makes sense to maintain the same format. Another (simpler) option would be to start using `switch` instead.

Reply:
Thanks for letting this know. It's my fault, in C language we can't use direct comparison in case of strings. I tried to replace all occurrences but I forget this one - thanks again.

Also in case of C language, we can't use `switch`-`case` structure for strings. The reason is that `case` expects a constant, not an array, and strings in C are arrays.

I replaced this line in 524a799.